README.md

```diff
@@ -108,9 +108,9 @@ Per Authorize.net's [Accept.js documentation](https://developer.authorize.net/ap
    };
 
    const App = () => {
-     const handleSubmit = (response) => {
+     const handleSubmit = React.useCallback((response) => {
```

Comment:
Thanks for the PR!

Why do we have to add `useCallback()` here? Correct me if I'm wrong but it's not strictly necessary, especially as (in your case I think?) this method actually does change based on application state e.g. some stateful value in your sidebar. I don't want people thinking that it's a function of the onSubmit prop that they *must* memoize it unless they actually have to.

Reply:
it's not strictly necessary but if you don't memoise callback responseHandler will be reinitialised on every re-render, and this will trigger the useEffect, which would re-assign the callback on window. So, it's better to memoising the callback.

Reply:
But isn't that what we want? In your case, doesn't intermediate state from, say, a sidebar component make changes to the handleSubmit method so that you'll need to re-instantiate the `window.responseHandler` callback? My point is, I don't want the docs adding unnecessary confusion when it's not necessary. Yes, an astute developer will realize that she may need to memoize the callback under certain circumstances, but memoizing it in the docs (with no dependencies!) adds confusion for me.

Reply:
In any case, please let me know if I'm totally off base here, it's been awhile since I was knee-deep in React. I'm going to merge this PR and probably revert this change, but I'll keep this thread open if I'm, like, totally wrong :)

Reply:
cool, then. I will remove it from docs